README.md

```diff
@@ -1,9 +1,17 @@
+---
```

Comment:
why did you restore the front matter in the readme file?

Reply:
> Sadly using include breaks all jekyll links inside the included part, reverting to using front matter in readme